modules/UI/videolayout/LargeVideoManager.js

```diff
@@ -26,6 +26,18 @@ const VIDEO_RESOLUTION_POLL_INTERVAL = 2000;
  * Manager for all Large containers.
  */
 export default class LargeVideoManager {
+    /**
+     * Checks whether given container is a {@link VIDEO_CONTAINER_TYPE}.
+     * FIXME currently this is a workaround for the problem where video type is
+     * mixed up with container type.
+     * @param {string} containerType
+     * @return {boolean}
+     */
+    static isVideoContainer(containerType) {
+        return containerType === VIDEO_CONTAINER_TYPE
+            || containerType === DESKTOP_CONTAINER_TYPE;
```

Comment:
This would be a bug fix for the connection interrupted message not displaying for desktop videos? Before only VIDEO_CONTAINER_TYPE was being checked but now VIDEO_CONTAINER_TYPE and DESKTOP_CONTAINER_TYPE are checked.